### PR TITLE
Use lodash templates

### PR DESCRIPTION
**Pull request by imurchie**
Instead of `swig`.

### CODE REVIEW

File: lib/static.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function guineaPig (req, res) {
     params.comment = req.body.comments || params.comment;
   }
   log.debug(`Sending guinea pig response with params: ${JSON.stringify(params)}`);
-  setTimeout(() => {
+  setTimeout(async () => {
```
Comment (jlipps) [inline lib/static.js]: would prefer to see:

``` js
if (delay) {
  await B.delay(delay);
}
```

File: lib/static.js [hunk 2]
```diff
@@ -33,21 +33,25 @@ function guineaPig (req, res) {
...
-  res.send(getTemplate('welcome.html')(params));
+  res.send((await getTemplate('welcome.html'))(params));
+  return 'done!';
 }
 
-function getTemplate (templateName) {
-  return swig.compileFile(path.resolve(STATIC_DIR, 'test', templateName));
+async function getTemplate (templateName) {
+  // swig-/mustache-like {{}} templating
+  _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
```
Comment (jlipps) [inline lib/static.js]: this is fine, or we could just switch the template to use lodash's `<%= content %>`.

**Comment by jlipps** [comment]
couple comments, like this a lot!

File: lib/static.js (hunk 1 above)
Comment (imurchie) [inline lib/static.js]: Good point. I'd prefer that too!

File: lib/static.js [hunk 3]
```diff
@@ -25,29 +26,29 @@ function guineaPig (req, res) {
...
-    res.set('Content-Type', 'text/html');
-    res.cookie('guineacookie1', 'i am a cookie value', {path: '/'});
-    res.cookie('guineacookie2', 'cookié2', {path: '/'});
-    res.cookie('guineacookie3', 'cant access this', {
-      domain: '.blargimarg.com',
-      path: '/'
-    });
-    res.send(getTemplate('guinea-pig.html')(params));
-  }, delay);
+  await B.delay(delay);
```
Comment (jlipps) [inline lib/static.js:29]: oh yeah and good point that if it's 0 we can still delay.

**Comment by jlipps** [comment]
this LGTM now

**Comment by SrinivasanTarget** [comment]
Looks good @imurchie

**Pull request merged**